beacon_node/http_api/src/publish_blocks.rs

```diff
@@ -110,13 +169,22 @@ pub async fn publish_block<T: BeaconChainTypes>(
             Ok(())
         }
         Err(e) => {
-            let msg = format!("{:?}", e);
-            error!(
-                log,
-                "Invalid block provided to HTTP API";
-                "reason" => &msg
-            );
-            Err(warp_utils::reject::broadcast_without_import(msg))
+            if let BroadcastValidation::Gossip = validation_level {
+                Err(warp_utils::reject::broadcast_without_import(format!(
+                    "{}",
+                    e
+                )))
```

Comment:
New-style format string would be good here

```suggestion
                Err(warp_utils::reject::broadcast_without_import(format!(
                    "{e}"
                )))
```